### DISCUSSION TITLE
How To Connect This To Monstercat Visualizer

### DISCUSSION THREAD

**Discussion by AilixiyaEden** [pear-devs/pear-desktop#2152] [Q&A]
Is there any way to connect this to Monstercat Visualizer?

**Comment by ArjixWasTaken** [comment]
Can you provide a link to that visualizer?

**Reply by AilixiyaEden** [reply]
Here is the link: https://github.com/marcopixel/monstercat-visualizer/releases/tag/2.1.0

**Reply by ArjixWasTaken** [reply]
There is no need for an integration to that project, as it is a rainmeter skin, therefore what you need is for the info to be accessible to rainmeter.

**Reply by ArjixWasTaken** [reply]
Try enabling the `shortcuts` and `taskbar-mediacontrol`plugins
